src/server/src/Server.cpp

```diff
@@ -132,6 +137,16 @@ bool Server::updateModule()
 
         set_state(State::PositionControl);
     }
+    /*
```

Comment:
Is this commented code a leftover? If you want to exclude this part of code, can you remove the comment? :)

Reply:
is already removed